server/user-bootstrap/index.js

```diff
@@ -20,6 +20,10 @@ module.exports = function( authCookieValue, callback ) {
 	if ( authCookieValue ) {
 		authCookieValue = decodeURIComponent( authCookieValue );
 
+		if ( typeof API_KEY !== 'string' ) {
+			throw new Error( 'Unable to boostrap user because of invalid API key in secrets.json' );
+		}
```

Comment:
I added this specific error has otherwise the error message thrown by the `crypto` module would be too cryptic (pun not indented, I promise) when `wpcom_calypso_rest_api_key` is not be provided.